README.md

```diff
@@ -2,21 +2,20 @@
 
 # Concepts in Computer Science: Algorithms
 
-
 ## Objectives
 
-By the end of this, developers should be able to:
+By the end of this, developers should be able to...
 
 -   Define "algorithm"
 -   Identify what Big-O time-complexity measures
 -   Give an example of a divide-and-conquer algorithm
 -   Predict the time-complexity of a given algorithm
 
-## Computer Science (0:05, 5 min)
+## Computer Science (5 minutes, 0:05)
 
```

Comment:
should we explicitly say what computer science is?

"Computer science is a discipline that involves the understanding and design of computers and computational processes. In its most general form it is concerned with the understanding of information transfer and transformation. Particular interest is placed on making processes efficient and endowing them with some form of intelligence. The discipline ranges from theoretical studies of algorithms to practical problems of implementation in terms of computational hardware and software..."

https://www.cs.mtu.edu/~john/whatiscs.html